src/models/dto/applicationStatus.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum ApplicationStatusEnum {
```

Comment:
can these be default exports? Also can we drop the `Enum` suffix to keep these consistent with our API? I think for enums you have to do:
```
enum ApplicationStatusEnum {
  ...
}

export default ApplicationStatusEnum
```